projects/nodejs/build.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -eu
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+cd node
+
+export LDFLAGS="$CXXFLAGS"
+export LD="$CXX"
+
+./configure --without-intl --without-node-code-cache --without-dtrace --without-snapshot --without-ssl
+make -j4
```

Comment:
Maybe change this to `-j$(nproc)` to get faster builds, when manually building this?

Reply:
Agreed